packages/commonwealth/client/scripts/views/components/UpvoteDrawer/ViewThreadUpvotesDrawer.tsx

```diff
@@ -28,7 +28,7 @@ export const ViewThreadUpvotesDrawer = ({
       avatarUrl: profile.avatarUrl,
       address: profile.address,
       updated_at: reactor?.updated_at,
-      voting_weight: reactor?.voting_weight || 0,
+      voting_weight: reactor?.voting_weight || 1,
```

Comment:
Hmm, this will indeed make it *look like* the user's vote has weight, but on the backend, their vote weight will still be zero.

I have a feeling we'll want to update it on the backend, so that'll automatically trickle down the change throughout all parts of the app?

Reply:
I'm doing this as a way of avoiding updating all of our users all at once. Some users may never buy stake. If they do, `voting_weight` will be updated. For those that don't, the value of their vote is 1. Per our Slack conversation, the plan according to @HIM92 is for users to have a default vote weight of 1.